Google.PowerShell/Logging/GcLogCmdlets.cs

```diff
@@ -90,6 +90,18 @@ protected string PrefixProject(string logName, string project)
         }
 
         /// <summary>
+        /// Prefix projects/{project id}/sinks to sinkName if not present.
+        /// </summary>
+        protected string PrefixProjectToSinkName(string sinkName, string project)
+        {
+            if (!string.IsNullOrWhiteSpace(sinkName) && !sinkName.StartsWith($"projects/{project}/sinks"))
+            {
+                sinkName = $"projects/{project}/sinks/{sinkName}";
```

Comment:
Hi @iantalarico, what do you mean by resources that will do this formatting? Like a .NET library?